recipients_app/lib/core/cubits/payment/payments_cubit.dart

```diff
@@ -22,9 +27,7 @@ class PaymentsCubit extends Cubit<PaymentsState> {
       emit(
```

Comment:
please try to extract separate calls and dont put it in emit. It makes it more readable and easier to debug:

```
final payments = await paymentRepository.fetchPayments(...);
final paymentUiState = await _mapPaymentsUiState(...);
emit(
  status: PaymentsStatus.success,
  payments: payments,
  paymentsUiState: paymentsUiState,
);
```

Reply:
I did fix that already